src/components/structures/MatrixChat.js

```diff
@@ -589,7 +589,16 @@ module.exports = React.createClass({
                     var theAlias = MatrixTools.getCanonicalAliasForRoom(room);
                     if (theAlias) presentedId = theAlias;
                 }
-                self.notifyNewScreen('room/'+presentedId);
+
+                if (typeof presentedId != 'undefined') {
```

Comment:
I was debugging issue: https://github.com/vector-im/vector-web/issues/1079
When a user open vector as guest, it is not defined.